index.html

```diff
@@ -0,0 +1,102 @@
+</head>
+<body>
+  <header class="mainMenu">
+    <img src="Icons/hamburger.svg" alt="menu" class="mobile menu">
+    <img src="Icons/Close.svg" alt="close" class="close mobile">
+    <div class="desktop">
+      <img src="Icons/top-logo.jpg" alt="logo">
+      <div class="menu">
+        <ul>
+          <li><a href="about.html" class="about">About</a></li>
+          <li><a href="#program-lineup">Program</a></li>
+          <li><a href="#guests">Speakers</a></li>
+          <li><a href="">Contacts</a></li>
+          <li><a href="#partners">Sponsors</a></li>
+        </ul>
+      </div>
+    </div>
+  </header>
+  <section class="first-section">
+    <div>
+      <p>"Hello! Everyone"</p>
+      <h1>MEETMENTORS EQUIP PROGRAM 2022</h1>
+      <div class="logo-div">
+        <img src="Icons/logo.jpg" alt="logo">
+      </div>
+      <div class="content">
+        <p>EQUIP is a talent and leadership focused training, designed to prepare young Africans to 
+          live intentionally and purposefully, while impacting the lives of others.</p>
+      </div>
+      <p class="date">2022.05.13 (FRI) ~ 15(SUN)</p>
+      <p class="meeting">Via Google meet... online</p>
+    </div>
+  </section>
+  <section class="program-lineup" id="program-lineup">
+    <h2>Program Line-up</h2>
+    <ul>
+      <li class="activity">
+        <div class="activity-title">Purpose</div>
+        <div class="activity-info">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quisquam et
+           consectetur provident tempora sit expedita, omnis praesentium nemo illum ex.</div>
+      </li>
+      <li class="activity">
+        <div class="activity-title">Vision</div>
+        <div class="activity-info">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quisquam et
+          consectetur provident tempora sit expedita, omnis praesentium nemo illum ex.</div>
+      </li>
+      <li class="activity">
+        <div class="activity-title">Career development</div>
+        <div class="activity-info">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quisquam et
+          consectetur provident tempora sit expedita, omnis praesentium nemo illum ex.</div>
+      </li>
+      <li class="activity">
+        <div class="activity-title">Leadership development</div>
+        <div class="activity-info">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quisquam et
+          consectetur provident tempora sit expedita, omnis praesentium nemo illum ex.</div>
+      </li>
+      <li class="activity">
+        <div class="activity-title">Building relationships</div>
+        <div class="activity-info">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quisquam et
+          consectetur provident tempora sit expedita, omnis praesentium nemo illum ex.</div>
+      </li>
+    </ul>
+    <div class="button-div"> <button>Join MeetMentors today</button> </div>
+  </section>
+  <section class="guests" id="guests">
+    
+  </section>
```

Comment:
There is an empty section tag, remove it.

Reply:
Hello @uche-inyama 
I left the IDs in the HTML file for the links in the header to be able to identify the segment. I however have removed all IDs from the CSS files

Reply:
@Elisha-Tetteyfio alright great work 👍🏾